File: Tandem Bicycle/O(nlogn)_time_and_O(1)_space.py
def tandemBicycle(redShirtSpeeds, blueShirtSpeeds, fastest):
    # Write your code here.
    redShirtSpeeds.sort()
    blueShirtSpeeds.sort(reverse= fastest)
    print(redShirtSpeeds,blueShirtSpeeds)
    sum = 0
    for ind in range(len(blueShirtSpeeds)):
        sum += max(blueShirtSpeeds[ind],redShirtSpeeds[ind] )
      
    return sum

File: Tandem Bicycle/O(nlogn)_time_and_O(1)_space.js
function tandemBicycle(redShirtSpeeds, blueShirtSpeeds, fastest) {
  // Write your code here.
  redShirtSpeeds.sort((a, b) => a - b);
  fastest
    ? blueShirtSpeeds.sort((a, b) => b - a)
    : blueShirtSpeeds.sort((a, b) => a - b);
  let sum = 0;
  for (let i = 0; i < blueShirtSpeeds.length; i++) {
    sum +=
      blueShirtSpeeds[i] > redShirtSpeeds[i]
        ? blueShirtSpeeds[i]
        : redShirtSpeeds[i];
  }
  return sum;
}
